test/FIFOMarket.test.ts

```diff
@@ -12,12 +12,42 @@ import {
 const setupTestLocal = async (
   {
     buyerInitialBPNoriBalance = formatTokenAmount(1_000_000),
-  }: { buyerInitialBPNoriBalance: BigNumberish } = {
+    nrtsToListAmounts = [],
+  }: {
+    buyerInitialBPNoriBalance?: BigNumberish;
+    nrtsToListAmounts?: number[];
```

Comment:
maybe change to `numberOfNrtsToList`

Reply:
or, rather than using a `startingVintage` below, just accept an array like:
```ts
{vintage: number, amountToList: number}[]
```

Reply:
ok cool, we're eventually going to need an abstraction around this other removal data, so even though it's really not necessary as part of this PR I'm refactoring this to take an array of objects, each of which can specify the details of a given removal that will be seeded into the marketplace.  everything but the amount field is optional, so that for tests where the supplier and vintage don't matter you don't always have to specify everything, these things are just assigned defaults in the setup function.

does allow me to further refactor one test whose setup had been left manual because it has to specify different suppliers